code/modules/mob/living/carbon/xenomorph/castes/spitter/abilities_spitter.dm

```diff
@@ -16,6 +16,8 @@
 	if(!istype(target)) //Something went horribly wrong. Clicked off edge of map probably
 		return
 
+	X.face_atom(target) //Face target so we don't look stupid
+
```

Comment:
Undo this, its better otherwise

Reply:
Wait, so you can just look away and shoot?

Reply:
Yes
Undo it

Reply:
Okay but that's silly asf